code/modules/mob/living/simple_animal/hostile/snork.dm

```diff
@@ -0,0 +1,22 @@
+//Inherits heavily from frog.dm
```

Comment:
they'll pick up spears, watch out for that (there's no sprite for a snork with spear)